ha/delete-cluster.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+virsh destroy node01
+virsh destroy node02
+virsh destroy node03
+
+virsh undefine node01 --remove-all-storage
+virsh undefine node02 --remove-all-storage
+virsh undefine node03 --remove-all-storage
```

Comment:
We should make this cleanup script idempotent so we can call it liberally to make sure the host is clean. I'd go with something like:
```
#!/bin/bash

VM_PREFIX="node0"

virsh list --state-running --name | grep "^$VM_PREFIX" | xargs -L 1 --no-run-if-empty virsh destroy
virsh list --state-shutoff --name | grep "^$VM_PREFIX" | xargs -L 1 --no-run-if-empty virsh undefine --remove-all-storage

# Check for leftover VMs (cleanup failed!)
! virsh list --all --name | grep -q "^$VM_PREFIX" || exit 1
```
with the PREFIX to be updated.